geode-dunit/src/main/java/org/apache/geode/test/dunit/VM.java

```diff
@@ -485,7 +485,15 @@ private synchronized void bounce(final String targetVersion, boolean force) {
       if (force) {
         processHolder.killForcibly();
       } else {
-        SerializableRunnableIF runnable = () -> new Thread(() -> System.exit(0)).start();
+        SerializableRunnableIF runnable = () -> new Thread(() -> {
+          try {
+            // sleep before exit so that the rmi call is returned
+            Thread.sleep(100);
```

Comment:
Was there an exception thrown in the controller? Since 100 still might not be enough on occasion, maybe we should catch the exception in the controller.